internal/logging/logging.go

```diff
@@ -112,12 +112,10 @@ func (lw *logWithWriters) WantQuiet(f bool) {
 }
 
 // NewLogWithWriters creates a logger to be used with pack CLI.
-func NewLogWithWriters() *logWithWriters {
+func NewLogWithWriters() *logWithWriters { //nolint:golint
```

Comment:
Can we ignore specific errors instead of all rules from a certain linter?

Reply:
Yes, I think. I'll try it. The doc says something like that is valid: `//nolint:golint,unused`

Reply:
We should fix this as well -- definitely better to export (or return through an interface).